server/src/main/java/org/elasticsearch/search/aggregations/bucket/significant/SignificantTermsAggregationBuilder.java

```diff
@@ -96,7 +97,7 @@
             aggregationParser.declareObject(SignificantTermsAggregationBuilder::significanceHeuristic,
                     (p, context) -> {
                         SignificanceHeuristicParser significanceHeuristicParser = significanceHeuristicParserRegistry
-                                .lookupReturningNullIfNotFound(name);
+                                .lookupReturningNullIfNotFound(name, LoggingDeprecationHandler.INSTANCE);
```

Comment:
I think this should be `p.getDeprecationHandler()`.

Reply:
Whoops yeah, I missed that `p` was a parser